style.css

```diff
@@ -0,0 +1,674 @@
+            }
+            footer span a{
+                color:var(--main); 
+            }
+            footer span{
+                margin: 0 2px;
+            }       
+
+
+
+ /*-------------
+            home media query
+                    -------------*/
+
+    @media(max-width:1015px){
+        .home-text{
+            margin-top: 30%;
+        }
+    }
+    @media(max-width:885px){
+        .home-text{
+            margin-top: 10%;
+
+        }
+        .home-text h1{
+            
+            font-size: 50px;
+            font-weight: 500;
+            
+    }
+        .home-text .text1{
+            text-align: center;
+             font-size: 30px;
+         }
+         .home-text .text2{
+             margin-left: 10px;
+            
+         }
+         .home-text .text3{
+             font-size: 20px;
+             font-weight: 200;
+             text-align: center;
+         }
+    } 
+    @media(max-width:700px){
+        .home-text h1{
+            
+            font-size: 40px;
+            font-weight: 500;
+            
+    } 
+    .home-text .text1{
+       text-align: center;
+        font-size: 20px;
+    }
+    .home-text .text2{
+        margin-left: 10px;
+       
+    }
+    .home-text .text3{
+        font-size: 20px;
+        font-weight: 200;
+        text-align: center;
+    }
+    
+}
+@media(max-width:450px){
+    #home .container{
+        flex-direction: column-reverse;
+    }
+    #home .right{
+        width: 100%;
+    }
+    #home .left{
+        width: 100%;
+    }
+}
+
+/*--------
+        home media query end
+        -------------------------*/
+/*nav bar*/
+@media screen and (max-width: 800px)
+    {
+        .container{
+            padding: 0 20px;
+            margin: 10px;
+        }
+        .menu-btn{
+            display: block;
+            z-index: 999;
+        }
+        .menu-btn i.active::before{
+            content: "\f00d";
+        }
+    .nav-bar .menu{
+        position: fixed;
+        background:var(--main);
+        height: 100vh;
+        width: 100%;
+        left: -100%;
+        top:0 ;
+        text-align: center;
+        padding-top: 60px;
+        transition: all 0.3s ease;
+        top: 0;
+    }
+    .nav-bar .menu.active{
+        left:0;
+    }
+    .nav-bar .menu li{
+        display: block;
+    }
+    .nav-bar .menu li a{
+        display: inline-block;
+        margin:20px 0;
+    }
+}
+
+ /*-------------
+            about media query
+                    -------------*/
+/*----------------
+          project media query
+           --------------------------*-*/
+  @media(max-width:700px){
+    .pr-item .left{
+        width: 100%;
+    }
+    .myprojects .pr-item{
+        flex-direction: column;
+    }
+    .pr-item .right{
+        width: 90%;
+        margin-top: -150px;
+        margin-left: 0;
+       
+       
+    }
+  }         
+
+
+
+
+/*--------------------
+            contact media query
+            ******************************/

```

Comment:
remove the extra * and --  -> /*  contact media query */ this is enough and please apply it everywhere